### ISSUE TITLE
Allow `Ctrl-C`, `Ctrl-X`, and `Ctrl-V`

### ISSUE THREAD

**Issue by Kully** [Kully/pixel-paint#35]
Upon selection, it would be useful to be able to use some of the more familiar operations that we have grown accustomed to.
-

**Comment by scriptorluminary** [comment]
Yes, for these operations implement in most programs, though I seldom hit the second. I type this to test.

**Comment by Kully** [comment]
> hello I am a student in Paris 8 and I need to develop an issue to validate. Can I work on it?

Yes go for it. Please open a Pull Request and request a review when it is ready.

**Comment by Morka951** [comment]
No , problem i gonna do that this week ! Please can you add the Licence in the readme

**Comment by Kully** [comment]
> No , problem i gonna do that this week ! Please can you add the Licence in the readme

Done. See [this commit](b93acf2fbe997106d2b8f2b49bc7db83c271b859).

Just make sure to pull from the `master` branch.

**Comment by Morka951** [comment]
👋 Hey!
I've opened a pull request to add support for the Ctrl+C, Ctrl+X, and Ctrl+V shortcuts for selection handling.
Could you please take a look and let me know if it follows the project's code conventions?
Feel free to share any suggestions or feedback. Thanks in advance! 😊
